### PR TITLE
fix(IV): use the initialize for the vault

### PR DESCRIPTION
**Pull request by kensodemann**
DO NOT MERGE: I am just looking for early comment on this from the React-facing SAs / DevSE.

### CODE REVIEW

File: demos/iv-ac/android/app/build.gradle [hunk 1]
```diff
@@ -1,6 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
+
+    packagingOptions {
+        resources {
+            excludes += ['META-INF/NOTICE', 'META-INF/LICENSE']
+        }
+    }
```
Comment (eric-horodyski) [inline demos/iv-ac/android/app/build.gradle:9]: Where did this change come from? Just curious, I have no idea who would have added this (a plugin, an updated version of Capacitor's Android template, Android Studio)

Comment (kensodemann) [reply demos/iv-ac/android/app/build.gradle:9]: I ran the gradle upgrade when testing in AS. I am guessing it came from there. Or perhaps from doing the build from AS.

Comment (kensodemann) [reply demos/iv-ac/android/app/build.gradle:9]: One way or another, it came from the tooling. I am just not sure _exactly_ where...

**Review by eric-horodyski — APPROVED** [review]
LGTM - At some point I would like to refactor the following code to use something provided by React out-of-the-box, instead of something flimsy and homebrewed.

```
type CallbackMap = {
  onSessionChange?: (session: AuthResult | undefined) => void;
  onVaultLock?: () => void;
  onPasscodeRequested?: (isPasscodeSetRequest: boolean, onComplete: (code: string) => void) => void;
};
```

I wonder if there's a way to do this with `useSyncExternalStore()`.

We'd be good on `vault.onPasscodeRequested()`, which just needs to handle the event on the React side, and can be set up using `subscribe` ([as seen here](https://react.dev/reference/react/useSyncExternalStore#subscribing-to-a-browser-api)). `onVaultLock` is a bit tricker because the local `session` object needs to be set to undefined during the event _and_ the event needs to have a subscriber on the React side to navigate to the unlock page. Not sure if one event handler would overwrite the other. Upside though, you'd get rid of `onSessionChange`, because `useSyncExternalStore` would update its snapshot eliminating the need to handle any sort of event.

**Comment by kensodemann** [comment]
> I wonder if there's a way to do this with `useSyncExternalStore()`.

I think that will be a good item to look into when we do the IV tutorials since those will be smaller projects where we can build that out and then use what we learn there to refactor the larger demos.

Also, it is really weird, IMO that GitHub apparently won't let me reply to your comment... grrrr....

**Pull request merged**